js/exporter/exceljs/export.js

```diff
@@ -47,6 +47,22 @@ export const Export = {
         excelCell.numFmt = numberFormat;
     },
 
+    getCellStyles: function(dataProvider) {
+        const styles = dataProvider.getStyles();
+
+        styles.forEach((style) => {
+            let numberFormat = this.tryConvertToExcelNumberFormat(style.format, style.dataType);
+
+            if(isDefined(numberFormat)) {
+                numberFormat = numberFormat.replace(/&quot;/g, '"');
+            }
+
+            style.numberFormat = numberFormat;
```

Comment:
Let's rename it to style.excelJSNumberFormat
dataProvider.getStyles is shared between exceljs and old exporting code